Lib/test/test_dis.py

```diff
@@ -1470,195 +1470,209 @@ def _prepare_test_cases():
+    ('LOAD_FAST'                 , 124,   2,                  'f',                       'f',   58,    8,  False),
+    ('RETURN_VALUE'              ,  83,   _,                 None,                        '',   60,    _,  False),
+]]
+
+expected_opinfo_f = [inst(*args) for args in [
+    # opname, opcode, arg, argval, argrepr, offset, starts_line, is_jump_target
+    ('COPY_FREE_VARS'            , 149,   2,                    2,                        '',    0,    _,  False),
+    ('MAKE_CELL'                 , 135,   0,                  'c',                       'c',    2,    _,  False),
+    ('MAKE_CELL'                 , 135,   1,                  'd',                       'd',    4,    _,  False),
+    ('RESUME'                    , 151,   0,                    0,                        '',    6,    2,  False),
+    ('LOAD_CONST'                , 100,   4,               (5, 6),                  '(5, 6)',    8,    3,  False),
+    ('LOAD_CLOSURE'              , 136,   3,                  'a',                       'a',   10,    _,  False),
+    ('LOAD_CLOSURE'              , 136,   4,                  'b',                       'b',   12,    _,  False),
+    ('LOAD_CLOSURE'              , 136,   0,                  'c',                       'c',   14,    _,  False),
+    ('LOAD_CLOSURE'              , 136,   1,                  'd',                       'd',   16,    _,  False),
+    ('BUILD_TUPLE'               , 102,   4,                    4,                        '',   18,    _,  False),
+    ('LOAD_CONST'                , 100,   3,    code_object_inner,   repr(code_object_inner),   20,    _,  False),
+    ('MAKE_FUNCTION'             , 132,   9,                    9,       'defaults, closure',   22,    _,  False),
+    ('STORE_FAST'                , 125,   2,              'inner',                   'inner',   24,    _,  False),
+    ('LOAD_GLOBAL'               , 116,   1,              'print',            'NULL + print',   26,    5,  False),
+    ('LOAD_DEREF'                , 137,   3,                  'a',                       'a',   38,    _,  False),
+    ('LOAD_DEREF'                , 137,   4,                  'b',                       'b',   40,    _,  False),
+    ('LOAD_DEREF'                , 137,   0,                  'c',                       'c',   42,    _,  False),
+    ('LOAD_DEREF'                , 137,   1,                  'd',                       'd',   44,    _,  False),
+    ('CALL'                      , 171,   4,                    4,                        '',   46,    _,  False),
+    ('POP_TOP'                   ,   1,   _,                 None,                        '',   56,    _,  False),
+    ('LOAD_FAST'                 , 124,   2,              'inner',                   'inner',   58,    6,  False),
+    ('RETURN_VALUE'              ,  83,   _,                 None,                        '',   60,    _,  False),
+]]
+
+expected_opinfo_inner = [inst(*args) for args in [
```

Comment:
Surround each of the formatted tables with `# fmt: off` and `# fmt: on` comments just in case we ever manage to auto-format the CPython codebase itself.

Reply:
done